report/assets/styles.css

```diff
@@ -280,6 +281,11 @@
   --section-padding-vertical: 8px;
 }
 
+.lh-container:not(.lh-topbar + .lh-container) {
```

Comment:
Worth noting this is "just for flow reports"?

double checked that PSI still has a lh-container (it does).

Reply:
Technically it's any renderer that passes `omitTopbar: true` so I don't think it's worth making a comment about how this is specific to flows.